templates/node/events/index.js

```diff
@@ -13,37 +15,27 @@ const { CloudEvent, HTTP } = require('cloudevents');
  *  -H'Ce-specversion: 1.0' \
  *  http://localhost:8080
  *
- * The event data is extracted from the incoming event and provided as the first
- * parameter to the function. The CloudEvent object itself may be accessed via the
- * context parameter, For example:
- *
- * const incomingEvent = context.cloudevent;
+ * For convenience, the runtime will extract the event from the
```

Comment:
```suggestion
 * For convenience, the runtime will extract the event data from the
```

Reply:
What does the sentence mean to you without the word "data"? What potential misinterpretation are you trying to prevent?